myFirstWebsite/index.html

```diff
@@ -118,13 +112,334 @@ <h2 class="benefits__item-title">
+                    </div>
+                    <div class="top-places__item">
+                        <div class="top-places__item-image-holder">
+                            <img src="../myFirstWebsite/images/top-places-6.jpg" alt="SHIBUYA" class="top-places__item-image">
+                            <div class="top-places__item-ui">
+                                <h3 class="top-places__item-title">
+                                    SHIBUYA
+                                </h3>
+                                <div class="top-places__item-likes">
+                                    <span class="likes-amount">228</span>
+                                    <img src="./images/like-icon.svg" alt="like" class="like-icon">
+                                </div>
+                                <div class="top-places__item-comments">
+                                    <span class="comments-amount">228</span>
+                                    <img src="./images/comment-icon.svg" alt="comment" class="comment-icon">
+                                </div>
+                            </div>
+                        </div>
+                        <p class="top-places__item-description">
+                            Jump off balcony, onto stranger's head. Chase ball of string hide when guests come over. Being gorgeous with belly side up i could pee on this.
+                        </p>
+                        <button type="button" class="top-places__more-btn common-btn">see more</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <div class="videos" id="videos">
+            <div class="videos__wrapper">
+                <div class="videos__adaptive-wrapper">
+                    <video controls width="1440" height="810" poster="./images/video-poster.jpg" >
+                        <source src="./videos/tokio_video.mp4">
+                        Your browser does not support the video tag.
+                    </video>
+                    <!-- <iframe src="https://www.youtube.com/embed/oik48bGBGBw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> -->
+                </div>
+            </div>
+        </div>
+        <footer class="footer">
+            <div class="footer__wrapper">
+                <div class="footer__logo">
+                    <img src="./images/site-logo.svg" alt="site logo" class="footer__logo-image">
+                </div>
+                <nav class="footer__nav">
+                    <ul class="footer__list">
+                        <li class="footer__item">
+                            <h3 class="footer__item-title">
+                                Company
+                            </h3>
+                        </li>
+                        <li class="header__item">
```

Comment:
Strange class name (header item in footer section). It's hard to change code in future. Check BEM

Reply:
+